purchase_work_acceptance/views/work_acceptance_views.xml

```diff
@@ -0,0 +1,175 @@
+                                   <field name="product_qty"/>
+                                   <field name="product_uom" groups="uom.group_uom" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
+                                   <field name="price_unit"/>
+                                   <field name="price_subtotal" widget="monetary" sum="Total"/>
+                               </tree>
+                               <form string="Work Acceptance Line">
+                                   <sheet>
+                                       <field name="state" invisible="1"/>
+                                       <group>
+                                           <group>
+                                               <field name="product_id"/>
+                                               <label for="product_qty"/>
+                                               <div>
+                                                   <field name="product_qty" class="oe_inline"/>
+                                                   <span class="oe_inline">&#160;</span>
+                                                   <field name="product_uom" groups="uom.group_uom" class="oe_inline"/>
+                                               </div>
+                                               <field name="price_unit"/>
+                                           </group>
+                                           <group>
+                                               <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                                           </group>
+                                       </group>
+                                   </sheet>
+                               </form>
+                           </field>
+                       </page>
+                   </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="view_work_acceptance_tree" model="ir.ui.view">
+        <field name="name">work.acceptance.tree</field>
+        <field name="model">work.acceptance</field>
+        <field name="arch" type="xml">
+            <tree string="Work Acceptance" decoration-muted="state=='cancel'" toolbar="1" >
+                <field name="name" string="Reference"/>
+                <field name="date_due" />
+                <field name="date_receive" />
+                <field name="date_accept" />
+                <field name="partner_id"/>
+                <field name="responsible_id"/>
+                <field name="company_id" groups="base.group_multi_company" options="{'no_create': True}"/>
+                <field name="origin"/>
+                <field name="currency_id" invisible="1"/>
+                <field name="state"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_work_acceptance_kanban" model="ir.ui.view">
```

Comment:
Let's remove kanban view for now. I think the tree view will do just fine now.
In future, we could have kanban, report, calendar and activity view also :)
Let's keep things simple, for this first version.